ngmix/tests/test_priors_base.py

```diff
@@ -34,4 +34,5 @@ def test_priors_gpriorbase_raises():
         with pytest.raises(RuntimeError) as e:
             args = tuple([None] * num)
             mth(*args)
-            assert "over-ride me" in e.value
+            # unreachable
+            assert "over-ride me" in e.value  # pragma: no cover
```

Comment:
If we can never get here then we don't need the test, right?

Reply:
right, I wasn't sure why you had that there at all since it should be unreachable